src/pages/SavedWorkflows.js

```diff
@@ -0,0 +1,101 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h, h3 } from 'react-hyperscript-helpers'
+import { AutoSizer } from 'react-virtualized'
+import { ButtonOutline } from 'src/components/common'
+import { FlexTable, Sortable, TextCell } from 'src/components/table'
+import * as Utils from 'src/libs/utils'
+
+
+export const SavedWorkflows = ({ setWorkflowUrl, setShowInputsPage }) => {
+  // State
+  const [sort, setSort] = useState({ field: 'submissionTimestamp', direction: 'desc' })
+
+  // this hardcoded data to be removed in https://broadworkbench.atlassian.net/browse/BW-1318
+  const previousRuns = [
+    {
+      workflowUrl: 'https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.wdl',
+      submissionTimestamp: '2022-01-27T22:27:15.591Z'
+    },
+    {
+      workflowUrl: 'https://raw.githubusercontent.com/abc/xyz',
+      submissionTimestamp: '2022-07-14T22:22:15.591Z'
+    },
+    {
+      workflowUrl: 'https://dockstore/abc/hello.wdl',
+      submissionTimestamp: '2022-07-12T22:57:15.591Z'
+    },
+    {
+      workflowUrl: 'https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.wdl',
+      submissionTimestamp: '2021-01-27T22:28:15.591Z'
+    },
+    {
+      workflowUrl: 'https://raw.githubusercontent.com/abc/xyz',
+      submissionTimestamp: '2022-07-14T22:28:15.591Z'
+    },
+    {
+      workflowUrl: 'https://dockstore/abc/hello.wdl',
+      submissionTimestamp: '2021-05-12T22:26:15.591Z'
+    },
+    {
+      workflowUrl: 'https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.wdl',
+      submissionTimestamp: '2022-01-27T22:29:15.591Z'
+    },
+    {
+      workflowUrl: 'https://raw.githubusercontent.com/abc/xyz',
+      submissionTimestamp: '2019-10-20T22:22:15.591Z'
+    },
+    {
+      workflowUrl: 'https://dockstore/abc/hello.wdl',
+      submissionTimestamp: '2019-07-12T22:24:15.591Z'
+    }
+  ]
+
+  const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, previousRuns)
+
+  return div({ style: { marginTop: '2em', display: 'flex', flexDirection: 'column' } }, [
+    h3(['Saved Workflows']),
```

Comment:
How would you feel about changing this to "Use a saved workflow"? I'm thinking about having it match the style of the existing "Provide a URL to a workflow", rather than one being an action phrase and the other being a descriptive title.

Reply:
I think what you are suggesting makes sense. But I slightly prefer "Saved Workflows" as this is a whole section where (I think, and it might change later) the user would be able to see both the workflows and inputs they submitted. So this will become a place also to view those results as well as reuse both. 

Re: adding `new` in "Provide a URL to a [new] workflow" - I am thinking this might confuse users that they can't reuse previously submitted workflows? But I could be wrong. 

I feel like leaving it as it for now. Although it might be worth bringing this to UX.